packages/aws-cdk/test/integ/cli/cdk-helpers.ts

```diff
@@ -178,7 +179,9 @@ export class TestFixture {
   }
 
   public async cdk(args: string[], options: CdkCliOptions = {}) {
-    return this.shell(['cdk', ...args], {
+    const verbose = options.verbose ?? true;
```

Comment:
Feels slightly extreme to turn on verbosity by default. How many extra lines does this print per invocation?

Logging can be expensive in terms of noise, cost in $, cost in developer time (scrolling back and forth on a web console or downloading) and execution latency (writing to stdout is not cheap). 
It's kind of like turning on debug logging in prod, but not that bad obviously.

Reply:
From Rico's pr I mentioned:

"Now that we suppress output of non-failing tests, it becomes
all the more important to have detailed information for failing tests."

The logs will only be printed out in the case of failure